.circleci/config.yml

```diff
@@ -43,6 +43,11 @@ build_and_test_defaults: &build_and_test_defaults
     - *build_step
     - *test_step
 
+check_dirty: &check_dirty
```

Comment:
Name this `check_dirty_step` and move it with the other build steps above. Also, remove the `- ` before `run` otherwise you won't be able to combine it with other steps.